README.md

```diff
@@ -1 +1,21 @@
-# SumoPY
\ No newline at end of file
+# Using Conda to Create Virtual Environments
+
+## Installing Anaconda 
+Install Anaconda to your computer using this link: https://www.anaconda.com/products/individual. Since this includes many packages that you may not need, if you would prefer to install only the things you need, you can install condo using this link: https://docs.conda.io/en/latest/miniconda.html
+
+## Creating and Activating Virtual Environments
+You can create a virtual environment via Anaconda navigator or command line. If you use Anaconda Navigator, simply go to the “Environments” tab on the left, and select “Create” to make a new environment. Name your environment and choose your packages accordingly. 
+
+## Using Command Line 
+If you would like to create and activate a virtual environment through command line, first check that your conda is up to date using: 
+`conda update conda`
```

Comment:
Markdown did not create a line break. I believe you need to insert an empty line. Also, can you add a tab (or 4 spaces) to create a code block? So instead of "to date using: `conda update conda`", try

"to date using:

<tab or 4 spaces>conda update conda

" 

Also can you do the same for the code lines below?